### ISSUE TITLE
[Feature Request] Ability to disable zooming

### ISSUE THREAD

**Issue by adityaravishankar** [tachiyomiorg/tachiyomi#4145]
## Why/User Benefit/User Problem
Currently, the slightest accidental gesture seems to keep causing unwanted zoom in/out which can get very annoying. Just a simple way to disable zoom gestures for people who don't want to use zoom can be very helpful. 

## What/Requirements

An on/off switch for zoom related gestures in the preferences>reader>navigation section, which can be used to disable pinch and double taps. 

Any other simple way of disabling zoom would also work. For example the way the Neko app allows disabling zoom for webtoons only)
![IMG_20201217_154028](https://user-images.githubusercontent.com/1101346/102474415-9bdde680-407e-11eb-9225-dc0fead0f2aa.jpg)

**Comment by scb261** [comment]
More general feature compared to #3918, not sure if it counts as duplicate

**Comment by vengiare** [comment]
"Enable zoom out" sounds really weird. Could probably use "Enable zoom/Allow zooming" and have it on by default instead.

**Comment by ylvc** [comment]
Please add this feature, in my opinion a simple toggle to enable/disable zoom out in webtoons would be great.

**Comment by leetdavid** [comment]
Yes, I think having a toggle to set the max width to 100% to prevent zooming out too much would be a great quality of life feature.

**Comment by CrimH** [comment]
Please enanble this feature. The zoomout is annoying when I read JP mangas. It zooms out too much. Please enable it.☹️

**Comment by Batmanerman** [comment]
Agreed on this feature, please allow us to disable zoom completely.

**Comment by lfuser** [comment]
If this goes ahead, I'd suggest being able to enable/disable each zooming gesture separately.
1) Pinch to zoom.
2) Double tap to fit to screen / zoom in.
3) Double tap and hold to zoom (moving up/down).
Disabling all forms of zooming would me this unusable.

In my case, I often trigger #3 unintentionally when I want to use #2 followed by scrolling in webtoon view, so disabling just #3 would be optimal.

Issue #4970 seems like user would like to disable both #2 and #3, but keep #1. This implementation solves both cases and possibly others.

**Comment by Nod0ff** [comment]
Please add this option. The number of times I unintentionally zoom, while scrolling in webtoon mode, is aggravating.

**Comment by Nyxeka** [comment]
Bumping this, surprised it's two years old. I don't know what's happened in the last little while but the zoom feature makes this app unusable in any scroll mode.

This is an extremely aggravating problem.

**Comment by Nod0ff** [comment]
> Bumping this, surprised it's two years old. I don't know what's happened in the last little while but the zoom feature makes this app unusable in any scroll mode.
> 
> This is an extremely aggravating problem.


Agreed completely. I get so tired of it auto zooming when I'm trying to scroll, then I need to manually zoom back in, and hope I don't over zoom, to get it to fit back perfectly.

**Comment by FireCheck** [comment]
Please add this feature quickly or you could simply copy it from TachiJ2K directly, it would be easier I think.
![IMG_20230313_051029.jpg](https://user-images.githubusercontent.com/84283461/224606375-8c2c1844-562d-4813-bc8a-7fc43c398d07.jpg)

**Comment by Nod0ff** [comment]
> Please add this feature quickly or you could simply copy it from TachiJ2K directly, it would be easier I think. ![IMG_20230313_051029.jpg](https://user-images.githubusercontent.com/84283461/224606375-8c2c1844-562d-4813-bc8a-7fc43c398d07.jpg)

Did not know about J2K, but that's such a massive QoL feature that we should already have.

**Comment by 101plusfourty** [comment]
How is this not added yet? It's great qol...

**Comment by asw3** [comment]
> If this goes ahead, I'd suggest being able to enable/disable each zooming gesture separately.
> 
>     1. Pinch to zoom.
> 
>     2. Double tap to fit to screen / zoom in.
> 
>     3. Double tap and hold to zoom (moving up/down).
> 
> 
> Disabling all forms of zooming would make this feature unusable.
> 
> In my case, I often trigger 3 unintentionally when I want to use 2 followed by scrolling in webtoon view, so disabling just 3 would be optimal.
> 
> Issue #4970 seems like user would like to disable both 2 and 3, but keep 1. This implementation solves both cases and possibly others.

Bumping to add to how needed disabling zooming by multiple tapping as an option is.

**Comment by Hippo0o** [comment]
will the varous annoying zoom gestures ever be toggleable or do i have to deal with them for another few years :(
